front_end/entrypoints/rn_fusebox/rn_fusebox.ts

```diff
@@ -131,9 +131,8 @@ if (globalThis.FB_ONLY__reactNativeFeedbackLink) {
         return false;
       }
 
-      Host.InspectorFrontendHost.InspectorFrontendHostInstance.openInNewTab(
```

Comment:
Hmm, I would prefer to keep this as `openInNewTab`. Would it make sense to add the `focus()` behaviour there?

https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/35aa264a622e853bb28b4c83a7b5cc3b2a9747bc/front_end/core/host/InspectorFrontendHost.ts#L168-L170

Reply:
Updated. Will observe if `focus()` has an effect once we've updated the URL.